test/integration/builder/inserts.js

```diff
@@ -702,6 +702,15 @@ module.exports = function(knex) {
         })
       });
 
+      it('transaction.batchInsert using specified transaction', function() {
+        return knex.transaction(function(tr) {
+          tr.batchInsert('BatchInsert', items, 30)
+          .returning(['Col1', 'Col2'])
+          .then(tr.commit)
+          .catch(tr.rollback);
```

Comment:
Any reason why we don't just simplify our transaction using test cases like this:

```
return knex.transaction(functinon(tr) {
    return tr.batchInsert('BatchInsert', items, 30);
});
```
Since we do not do anything special with the result, there seems to be no need to call `returning()` + explicitly calling `tr.commit()` & `tr.rollback()` just seems like potentially confusing noise.

Though I see this is pretty much a copy/paste of an already existing test so we could do that in a separate cleanup pull request as well.

@alexwhitman, @wubzz - what do you think?